tests/dialects/test_postgres.py

```diff
@@ -233,6 +233,14 @@ def test_postgres(self):
                 "tsql": "SELECT * FROM t CROSS JOIN GENERATE_SERIES(2, 4)",
             },
         )
+        self.validate_all(
+            "SELECT * FROM t CROSS JOIN GENERATE_SERIES(2, 4) AS s",
+            write={
+                "postgres": "SELECT * FROM t CROSS JOIN GENERATE_SERIES(2, 4) AS s",
+                "presto": "SELECT * FROM t CROSS JOIN UNNEST(SEQUENCE(2, 4)) AS _u(s)",
+                "trino": "SELECT * FROM t CROSS JOIN UNNEST(SEQUENCE(2, 4)) AS _u(s)",
+            },
```

Comment:
I didn't add a `tsql` test case here, because I'm not familiar with it, so any pointers would be helpful!

Reply:
For `tsql` we should just transpile the original query back to itself:

<img width="1097" alt="Screenshot 2023-05-12 at 4 34 24 PM" src="https://github.com/tobymao/sqlglot/assets/46752250/030fc7a3-fdad-4e27-8df6-b96eb7c45205">

Reply:
What's this site? seems handy!

Reply:
It's https://onecompiler.com/sqlserver